src/Microsoft.PowerShell.Commands.Management/commands/management/GetComputerInfoCommand.cs

```diff
@@ -558,13 +558,13 @@ private static MiscInfoGroup GetOtherInfo(CimSession session)
             // get secure-boot info
             //TODO: Local machine only? Check for that?
             FirmwareType fwType = FirmwareType.Unknown;
-            if (Native.GetFirmwareType(ref fwType))
+            if (GetFirmwareType(ref fwType))
```

Comment:
The basic pattern `Get-ComputerInfo` uses is that for any given property in the returned object if the cmdlet was unable to acquire the value then that property remains null. Setting, for example, the `BiosFirmwareType` to `FirmwareType.Unknown` when we fail to acquire the actual value would deviate from that pattern.

Setting a default value for `CsPhysicallyInstalledMemory` would be a worse deviation from that pattern. In that case we would be telling the user that the computer has 0 bytes of installed memory, which is clearly not true. By leaving the property null we are saying that we were not able to acquire the correct value for that property and chose not to provide a misleading value.